test/test_prototype_builtin_datasets.py

```diff
@@ -109,19 +110,39 @@ def test_transformable(self, test_home, dataset_mock, config):
 
         next(iter(dataset.map(transforms.Identity())))
 
-    @pytest.mark.xfail(reason="See https://github.com/pytorch/data/issues/237")
     @parametrize_dataset_mocks(DATASET_MOCKS)
-    def test_serializable(self, test_home, dataset_mock, config):
+    def test_traversable(self, test_home, dataset_mock, config):
         dataset_mock.prepare(test_home, config)
+        dataset = datasets.load(dataset_mock.name, **config)
 
+        traverse(dataset, only_datapipe=False)
```

Comment:
Should we parametrize over `only_datapipe` or would this be overkill?

Reply:
@NicolasHug you are right. Let's be safe here.

@ejguan The test are already failing until your PR is merged so we need to wait anyway.